public/app/features/dashboard/components/PanelEditor/state/selectors.ts

```diff
@@ -35,7 +37,9 @@ export const getPanelEditorTabs = memoizeOne((tab?: string, plugin?: PanelPlugin
   }
 
   if (
```

Comment:
Suggestion: Can we split these into  a couple of variables? I think that would make it a bit easier to reason about the boolean logic.

```js
const { alertingEnabled, unifiedAlertingEnabled } = getConfig();
const hasReadPermissions = contextSrv.hasPermission(getRulesPermissions('grafana').read);

const isGraph = plugin.meta.id === 'graph';
const isTimeseries = plugin.meta.id === 'timeseries'

if (alertingEnabled || (unifiedAlertingEnabled && hasReadPermissions && isGraph) || isTimeseries) { ... }
```

Reply:
Good point. I'll update